### PR TITLE
Script now creates (if need be) /usr/local/etc and /usr/local/etc/profile.d, and links (if need be) /etc/profile to /usr/local/etc/profile (for bash support) and symlinks GNUstep.sh to /usr/local/etc/profile.d/GNUstep.sh, so that it will be automatically sourced by sh and bash.

### PR DESCRIPTION
**Pull request by KyleCardoza**

### CODE REVIEW

**Review by eukara — APPROVED** [review]
Should we not check explicitly for `/usr/local/etc/profile.d` existing, in case `/usr/local/etc` happens to exist on a non-stock system? It's probably fine for the time being though.

**Comment by KyleCardoza** [comment]
Yeah, I can see the reasoning there -- while the eventual goal is to have a custom respun FreeBSD-based installer image for VMs and real hardware, for *now* it's probably better to enable easier installation on existing FreeBSD installations. I'll get that in soon.

File: install-agora-git.sh [hunk 1]
```diff
@@ -182,4 +182,9 @@ nohup.out
 GNUstep
 EOF
 
+
+[ -x /usr/local/etc ] || (sudo mkdir -p /usr/local/etc/profile.d)
+[ -x /usr/local/etc/profile ] || (sudo ln -sf /etc/profile /usr/local/etc/profile)
+[ -x /usr/local/etc/profile.d/GNUstep.sh ] || (sudo ln -sf `gnustep-config --variable=GNUSTEP_MAKEFILES`/GNUstep.sh /usr/local/etc/profile.d/GNUstep.sh)
```
Comment (tedchoward) [inline install-agora-git.sh:188]: `-x` checks that the target is executable. I think you want `-d` for the directory, and `-f` for the files/links.

```suggestion
[ -d /usr/local/etc ] || (sudo mkdir -p /usr/local/etc/profile.d)
[ -f /usr/local/etc/profile ] || (sudo ln -sf /etc/profile /usr/local/etc/profile)
[ -f /usr/local/etc/profile.d/GNUstep.sh ] || (sudo ln -sf `gnustep-config --variable=GNUSTEP_MAKEFILES`/GNUstep.sh /usr/local/etc/profile.d/GNUstep.sh)
```

https://en.wikibooks.org/wiki/Bourne_Shell_Scripting/Control_flow

**Review by tedchoward — CHANGES_REQUESTED** [review]

File: install-agora-git.sh (hunk 1 above)
Comment (tedchoward) [reply install-agora-git.sh:188]: DOH, too late

Comment (tedchoward) [reply install-agora-git.sh:188]: #23

Comment (KyleCardoza) [reply install-agora-git.sh:188]: Accepted with thanks

**Pull request merged**